### PR TITLE
add_send_text_message

### PR DESCRIPTION
**Pull request by venkatram-dev**
To send text messages to the assistant aside from the audio input using the send method and passing appropriate role and content.

```

vapi = Vapi(api_key=key)

vapi.start(assistant_id='xxx')

time.sleep(5)

vapi.add_message('user', 'Can you explain VAPI in one sentence?')

time.sleep(15)

vapi.stop()


```

We can look at few results here 

https://dashboard.vapi.ai/calls/e40c730f-87dc-42a1-82f0-bce53697150c

https://dashboard.vapi.ai/calls/dd35f893-99dd-4366-a2a8-23c9ac04c5c9

Here is a screenshot of the message from vapi call logs.

Screenshot 2024-08-21 at 9 38 58 PM

### CODE REVIEW

File: vapi_python/vapi_python.py [hunk 1]
```diff
@@ -62,3 +62,21 @@ def start(
     def stop(self):
         self.__client.leave()
         self.__client = None
+
+    def send(self, message_type, message_content):
```
Comment (nikhilro) [inline vapi_python/vapi_python.py:66]: could you make the interface more generic `send(self, message)`

here's the behaviour from [web SDK](https://github.com/VapiAI/web/blob/cb7f315e49889c84e1b03bfaa1664d14d6b62608/vapi.ts#L319C2-L321C4)

```
 send(message: VapiClientToServerMessage): void
 ```

**Review by nikhilro — CHANGES_REQUESTED** [review]
thanks! mostly looks good, one comment above

File: vapi_python/vapi_python.py (hunk 1 above)
Comment (venkatram-dev) [reply vapi_python/vapi_python.py]: Thanks. Made the changes. I guess ,we can add the say and control functions later as I am not sure how to test them correctly :)

**Review by nikhilro — APPROVED** [review]

**Comment by nikhilro** [comment]
thanks!

**Pull request merged**